model/clusters_mgmt/v1/cluster_type.model

```diff
@@ -190,4 +190,7 @@ class Cluster {
 
 	// Channel group to be used for upgrading the cluster. Valid values: stable (default), fast, candidate, nightly.
 	UpgradeChannelGroup String
+
+	// Billing model for cluster resources. Valid values: standard (default), marketplace.
+	BillingModel String
```

Comment:
Is there any reason to not use an _enum_ type for this?

Reply:
No. I'll fix it